### DISCUSSION TITLE
SECTION 2

### DISCUSSION THREAD

**Discussion by vycod** [Cyfrin/moccasin-full-course-cu#22] [General]
Hey,
In **section 2** after the **21. Encrypt** I think we are missing the decrypt part,
i only saw a refferance to sign transaction decryption in the recap part
did i missed it?

**Comment by s3bc40** [comment]
Hey @vycod! So yes I had the same reaction, but I think I dug into the documentation of `web3.py` and `eth_account` when I had reached the workshop.

You'll find the method `decrypt` here:
https://eth-account.readthedocs.io/en/stable/eth_account.html 

Here is the link of my repo and how I have done it if you want to double check:

https://github.com/s3bc40/web3py-favs-cu/blob/39b9bb0e7f4e895935d00b545e04a2a1a9454222/deploy_favorites.py#L76-L84

**Reply by vycod** [reply]
yeh i saw it, thanks

**Reply by EngrPips** [reply]
Awesome.
